.github/workflows/code_quality.yaml

```diff
@@ -82,6 +82,9 @@ jobs:
       - run: poetry run pyright
         if: "!cancelled()"
 
+      - run: poetry run pyright
```

Comment:
I think we don't need it since we have the same code in lines # 82-83

Reply:
🤦🏽 thanks, I'm blind